indra/llcorehttp/_httplibcurl.cpp

```diff
@@ -487,12 +487,22 @@ void HttpLibcurl::policyUpdated(unsigned int policy_class)
         policy.stallPolicy(policy_class, false);
         mDirtyPolicy[policy_class] = false;
 
-        if (options.mPipelining > 1)
+        if (options.mPipelining)
         {
             // We'll try to do pipelining on this multihandle
-            check_curl_multi_setopt(multi_handle,
-                                     CURLMOPT_PIPELINING,
-                                     1L);
+            LL_INFOS("HttpLibcurl") << "HTTP Pipelining Enable." << LL_ENDL;
```

Comment:
Is this debug logging left in?  Should it not use the supplied LOG_* tag values above?